tests/admin/test_system_queries.py

```diff
@@ -86,10 +87,15 @@ def test_is_query_select() -> None:
         ("system KILL", False),
         ("ALTER TABLE eap_spans_local_merge DROP PARTITION '1970-01-01'", True),
         ("CREATE TABLE eap_spans_local_merge (all my fieds)", True),
+        ("OPTIMIZE TABLE eap_spans_local", True),
```

Comment:
Niit: Should we consider adding test case for 
Multiple spaces between words ("OPTIMIZE    TABLE    my_table")
Mixed case variations ("optimize table my_table")